superset/assets/src/explore/controls.jsx

```diff
@@ -2026,6 +2029,13 @@ export const controls = {
     description: t('Extra parameters for use in jinja templated queries'),
   },
 
+  time_range_endpoints: {
+    type: 'HiddenControl',
+    label: t('Time range endpoints'),
+    hidden: true,
+    description: t('Time range endpoints (SIP-15)'),
```

Comment:
Should we elaborate more about SIP-15 here? Provide a link or something?

Reply:
@etr2460 this control is always hidden and only exists to ensure that associated form-data is preserved.